keyboards/planck/keymaps/am/keymap.c

```diff
@@ -0,0 +1,266 @@
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+// this is the style you want to emulate.
+
+#include "planck.h"
```

Comment:
Could you replace all of the includes with just `#include QMK_KEYBOARD_H`, as that should cover all of these, actually.